app/task_routes.py

```diff
@@ -0,0 +1,157 @@
+from datetime import datetime
+from flask import Blueprint, jsonify, make_response, request, abort
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from sqlalchemy import desc #(used in read_all_tasks for sorting)
+import requests
+import os 
+# from slackclient import SlackClient #Alternate method 3 (below to call external api)
+
+SLACK_TOKEN = os.environ.get('SLACK_TOKEN', None)
+# slack_client = SlackClient(SLACK_TOKEN) #Alternate method 3
+
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+#CREATE Routes (Wave 1: CRUD Routes)
+@tasks_bp.route("", methods=["POST"])
+def create_task():
+        request_body = request.get_json()
+
+        try:
+            new_task = Task.from_dict(request_body)
+        except KeyError:
+            return jsonify ({"details": "Invalid data"}), 400
```

Comment:
Nice use of a try-except, but it would be good to include details about what data is invalid for the user.